js/main.js

```diff
@@ -4,21 +4,53 @@
 var $photoDisplay = document.querySelector('#photo-display');
 var $urlInput = document.querySelector('#url');
 var $form = document.querySelector('#input-form');
-var $entries = document.querySelector('ul.container');
+var $entriesContainer = document.querySelector('ul.container');
 var $tabsContainer = document.querySelector('#tabs');
 var $tab = document.querySelectorAll('.tab');
 var $view = document.querySelectorAll('.view');
 var $newEntryButton = document.querySelector('#new-entry-button');
+var editedEntryId = 0;
```

Comment:
I think you can avoid needing this variable at all if you use `data.editing.entryId` wherever you intended to use this variable.